packages/remediations/src/steps/reviewSystems.js

```diff
@@ -0,0 +1,166 @@
+import React, { useRef, useEffect, useState, Fragment } from 'react';
+import propTypes from 'prop-types';
+import useFieldApi from '@data-driven-forms/react-form-renderer/dist/esm/use-field-api';
+import useFormApi from '@data-driven-forms/react-form-renderer/dist/esm/use-form-api';
+import {
+    Text,
+    TextContent,
+    Stack, StackItem
+} from '@patternfly/react-core';
+import { InventoryTable } from '@redhat-cloud-services/frontend-components/Inventory';
+import { Provider, useDispatch, useSelector } from 'react-redux';
+import promiseMiddleware from 'redux-promise-middleware';
+import { BrowserRouter as Router } from 'react-router-dom';
+import ReducerRegistry from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { EXISTING_PLAYBOOK, inventoryEntitiesReducer as entitiesReducer, TOGGLE_BULK_SELECT } from '../utils';
+import './reviewSystems.scss';
+
+const ReviewSystems = (props) => {
+    const { issues, systems, registry } = props;
+
+    let dispatch = useDispatch();
+    const inventory = useRef(null);
+    const { input } = useFieldApi(props);
+    const formOptions = useFormApi();
+
+    const [ inventoryApi, setInventoryApi ] = useState();
+
+    const { selected, loaded, rows } = useSelector(({ entities }) => ({
+        selected: entities?.selected || new Map(),
```

Comment:
Map should not be used with reducers as it is a mutable construction. Can we use plain object instead?

Reply:
Changed to a basic array, Map is actually not needed